src/components/thumbnail/thumbnail-document-item.tsx

```diff
@@ -86,7 +86,7 @@ export const ThumbnailDocumentItem: React.FC<IProps> = observer((props: IProps)
         }
       </div>
       {
-        onDocumentStarClick &&
+        onDocumentStarClick && !user.isResearcher &&
```

Comment:
I think this will hide the bookmarks entirely. The ideal solution would be showing them but not making them editable. But if that is hard it isn't worth it.